pilot/util/harvester.py

```diff
@@ -145,11 +154,14 @@ def get_worker_attributes_file(args):
     :param args: Pilot arguments object.
     :return: worker attributes file name.
     """
+
+    logger.debug('config.Harvester.__dict__ : {0}'.format(config.Harvester.__dict__))
+
     if args.harvester_workdir != '':
         work_dir = args.harvester_workdir
     else:
         work_dir = environ['PILOT_HOME']
-    worker_attributes_file = config.Harvester.workerAttributesFile
+    worker_attributes_file = config.Harvester.workerattributesfile
     worker_attributes_file = join(work_dir, worker_attributes_file)
```

Comment:
did you mean os.path.join?

Reply:
yes changed to os.path.join for readability